Common/include/option_structure.inl

```diff
@@ -1696,7 +1696,22 @@ class COptionActDisk : public COptionBase {
     this->name = name;
   }
 
-  ~COptionActDisk() override{};
+  ~COptionActDisk() override {
+    for (int i = 0; i < this->inlet_size; i++) {
+      delete[] this->press_jump[i];
+      delete[] this->temp_jump[i];
+      delete[] this->omega[i];
+    }
+    delete[] press_jump;
+    delete[] temp_jump;
+    delete[] omega;
+
+    delete[] marker_inlet;
+    delete[] marker_outlet;
+
+    SetDefault();
+  };
```

Comment:
Indeed, `string *Marker_ActDiskBemInlet, *Marker_ActDiskBemInlet` were accessed via both the config file options `MARKER_ACTDISK_BEM_CG`, `MARKER_ACTDISK_BEM_AXIS`, and thus `delete[]`'d by both COptionActDisk instances. This was not a bug because `SetDefault` sets the pointers to zero.

As this is still kind of a dangerous construct (i.e. if the user supplies different marker lists to both cfg file options, one of them "wins" without an error message), [fc23554](https://github.com/su2code/SU2/pull/2212/commits/fc2355455e679b15ec0b8ab028a17e1df93af112) refactors the code such that both options initialize different marker lists, and those are checked to be identical in `CConfig::SetPostprocessing`.